<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogIn</class>
 <widget class="QWidget" name="LogIn">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>457</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{background-color: rgb(154, 217, 245)}
QPushButton:hover{background-color: rgb(0, 204, 255)}
QPushButton:pressed{background-color: rgb(71, 163, 215)}
</string>
  </property>
  <widget class="QWidget" name="loginwidget" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="loginbtn">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>350</y>
     <width>331</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 22px; </string>
   </property>
   <property name="text">
    <string>登 录</string>
   </property>
  </widget>
  <widget class="QLabel" name="LoginEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>401</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 20pt &quot;黑体&quot;;
</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt;&quot;&gt;系统账号登录&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="closeBtn">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>10</y>
     <width>28</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/bin/images/btn_close_n.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="wrongEdit2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>320</y>
     <width>331</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;HarmonyOS Sans SC&quot;;
color: rgb(255, 60, 43);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="pwdlabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>230</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;HarmonyOS Sans SC&quot;;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;密码&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="namelabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>140</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;HarmonyOS Sans SC&quot;;
</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;账户&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="linePassWord">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>270</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineName">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>180</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HarmonyOS Sans SC</family>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 16px;
font-weight: bold;</string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelLog">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>420</y>
     <width>20</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 24pt ;</string>
   </property>
   <property name="text">
    <string>|</string>
   </property>
  </widget>
  <widget class="QWidget" name="backwidget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>510</y>
     <width>421</width>
     <height>511</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="minisizeBtn">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>28</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/bin/images/btn_min_n.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="chatBtn">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>430</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 16px;</string>
   </property>
   <property name="text">
    <string>联系客服</string>
   </property>
  </widget>
  <widget class="QPushButton" name="signBtn">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>430</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 16px;</string>
   </property>
   <property name="text">
    <string>注册账号</string>
   </property>
  </widget>
  <widget class="QPushButton" name="forgetBtn">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>140</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 16px;</string>
   </property>
   <property name="text">
    <string>清空所有</string>
   </property>
  </widget>
  <zorder>backwidget</zorder>
  <zorder>loginwidget</zorder>
  <zorder>loginbtn</zorder>
  <zorder>LoginEdit</zorder>
  <zorder>closeBtn</zorder>
  <zorder>wrongEdit2</zorder>
  <zorder>pwdlabel</zorder>
  <zorder>namelabel</zorder>
  <zorder>linePassWord</zorder>
  <zorder>lineName</zorder>
  <zorder>labelLog</zorder>
  <zorder>minisizeBtn</zorder>
  <zorder>chatBtn</zorder>
  <zorder>signBtn</zorder>
  <zorder>forgetBtn</zorder>
 </widget>
 <tabstops>
  <tabstop>lineName</tabstop>
  <tabstop>linePassWord</tabstop>
  <tabstop>loginbtn</tabstop>
  <tabstop>signBtn</tabstop>
  <tabstop>chatBtn</tabstop>
  <tabstop>forgetBtn</tabstop>
  <tabstop>closeBtn</tabstop>
  <tabstop>minisizeBtn</tabstop>
  <tabstop>wrongEdit2</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
